src/modules/commands/missionAdd.js

```diff
@@ -13,7 +14,44 @@ function getDiscordIdFromMention(mention) {
     return match && match[1];
 }
 
+// todo: move it somewhere | prbbly to checker. Unification
+function modifySettings(originalSettings, checker, missionId) {
+    if (checker === 'linkChecker') {
+        // eslint-disable-next-line no-param-reassign
+        originalSettings.missionId = missionId;
+    }
+
+    return originalSettings;
+}
+
+// todo: move it somewhere | prbbly to checker. Unification
+function modifyDescriptionValues(values, checker, missionId, assignee) {
```

Comment:
its part of checker _INIT_ section... or something like this...

probably, i think, all checkers its linear modules with submodule "checker"... like command.filter module =)